src/Sulu/Bundle/WebsiteBundle/Twig/Navigation/NavigationTwigExtensionInterface.php

```diff
@@ -74,4 +74,14 @@ public function flatNavigationFunction($uuid, $context = null, $depth = 1, $load
      * @return \Sulu\Bundle\WebsiteBundle\Navigation\NavigationItem[]
      */
     public function breadcrumbFunction($uuid);
+
+    /**
+     * Returns a boolean value to check if navigation item is active.
+     *
+     * @param string $requestUrl
+     * @param string $itemUrl
+     *
+     * @return \Sulu\Bundle\WebsiteBundle\Navigation\NavigationItem[]
+     */
+    public function isActiveNavElementFunction($requestUrl, $itemUrl);
```

Comment:
this interface is also implemented in the class `MemoizedNavigationTwigExtension`